stats_test.go

```diff
@@ -102,6 +104,63 @@ func TestSubmitMetricError(t *testing.T) {
 		t.Errorf("Expected an error")
 	}
 }
+
+func TestSubmitCount(t *testing.T) {
```

Comment:
this fails with previous implementation
output is as such:
```
		`fooCount:1|g`,
		`fooCount:1|g`,
```
was gauge, persistent 1 value